### PR TITLE
Adding previous 'ServerCrypto' class with modifications done to work …

### PR DESCRIPTION
**Pull request by Pushpalanka**
…with latest wss4j version. (Note: XKMS related code segments are removed as we no longer support it.)

### CODE REVIEW

**Comment by callkalpa** [comment]
Hi @Pushpalanka,

This need to have a Carbon JIRA and the fix needs to be reviewed by the team lead.

**Comment by Pushpalanka** [comment]
Cloned the jira at IDENTITY 'https://wso2.org/jira/browse/IDENTITY-3546' and moved to WSAS jira 'https://wso2.org/jira/browse/WSAS-2150
'

File: components/webapp-mgt/org.wso2.carbon.cxf.ext/pom.xml [hunk 1]
```diff
@@ -30,6 +30,15 @@
...
             <artifactId>javax.ws.rs-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.ws.security</groupId>
+            <artifactId>wss4j</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity</groupId>
+            <artifactId>org.wso2.carbon.security.mgt</artifactId>
+            <version>${carbon.identity.version}</version>
```
Comment (johannnallathamby) [inline components/webapp-mgt/org.wso2.carbon.cxf.ext/pom.xml]: shouldn't we add version at root level ?

**Comment by johannnallathamby** [comment]
Reviewed. Can merge.

**Comment by Pushpalanka** [comment]
Updated the pom.xml to remove the version definition at this level as pointed out by Johann..

**Pull request merged**